drivers/kw2xrf/kw2xrf.c

```diff
@@ -104,8 +104,6 @@ void kw2xrf_reset_phy(kw2xrf_t *dev)
 
     kw2xrf_set_rx_watermark(dev, 1);
 
-    kw2xrf_set_option(dev, KW2XRF_OPT_AUTOACK, true);
-    kw2xrf_set_option(dev, KW2XRF_OPT_ACK_REQ, true);
     kw2xrf_set_option(dev, KW2XRF_OPT_AUTOCCA, true);
```

Comment:
I kept the `AUTOCCA` flag here as it has a slightly different meaning than the `CSMA` netopt. It also has its own netopt option.

Reply:
An alternative could be to try setting csma first, and if that fails, try to set autocca. What do you think of this?

Reply:
We can copy the behavior from kw41zrf. I did exactly what you describe in order to provide csma for that radio.

Reply:
Someone else will have to provide the PR. I don't have the hardware and have no interest in the kw2xdrf

Reply:
> Someone else will have to provide the PR. I don't have the hardware and have no interest in the kw2xdrf

Same for me to be honest

Reply:
Well, we're all short in time but we (HAW Hamburg) have certain interest in that radio so it might be us to do the follow-up. @gebart, @bergzand could you please open a feature request issue?

Reply:
@PeterKietzmann I opened #10364 as a reminder